Fix: Navbar style and grammar improvements

rajasmitaa opened the pull request:
This PR fixes issue #28 

1. Updated navbar color to match brand palette across all pages.

2. Adjusted padding for better spacing and visual consistency.

3. Corrected grammatical issues in the Masthead page.

rajasmitaa commented:
Hi @Kritika75 , I've resolved the conflicts and added the required changes.  
Here are the screenshots of the updated navbar across pages, before vs after:

**Improved colour of navbar:**

Before:

<img width="1919" height="959" alt="Screenshot 2025-07-23 120948" src="https://github.com/user-attachments/assets/98b045af-7966-42ab-b3c5-e714f7810888" />
<img width="1919" height="913" alt="Screenshot 2025-07-23 121004" src="https://github.com/user-attachments/assets/b3a4a9ea-d5a4-4889-97c6-1ede4ae4d20e" />

After:

<img width="1919" height="1012" alt="Screenshot 2025-07-23 121823" src="https://github.com/user-attachments/assets/f3198e5a-13e5-4819-8fca-bd9cfde825a3" />
<img width="1916" height="932" alt="Screenshot 2025-07-23 121805" src="https://github.com/user-attachments/assets/d0d37c06-0947-4970-9d0f-f189834d33a7" />

**Fixed grammatical errors( found ---> find):**

Before:

<img width="1919" height="954" alt="Screenshot 2025-07-23 121032" src="https://github.com/user-attachments/assets/e0f4cb9c-1136-4c6f-9208-ff83afc0673b" />

After:

<img width="1919" height="1011" alt="Screenshot 2025-07-23 121748" src="https://github.com/user-attachments/assets/73546994-f603-44f6-8394-c62f3cafa670" />



Let me know if anything else needs to be updated. Thank you!

Kritika75 commented:
> Hi @Kritika75 , I've resolved the conflicts and added the required changes. Here are the screenshots of the updated navbar across pages, before vs after:
> 
> **Improved colour of navbar:**
> 
> Before:
> 
> <img alt="Screenshot 2025-07-23 120948" width="1919" height="959" src="https://private-user-images.githubusercontent.com/146310937/469600035-98b045af-7966-42ab-b3c5-e714f7810888.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAwMDM1LTk4YjA0NWFmLTc5NjYtNDJhYi1iM2M1LWU3MTRmNzgxMDg4OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Mzc2ZjFjNWMxYWIxNzIwODUwNDg2Mzk2NWZlYTEzZjU2MWU4OTFhNGQxYzVkMTY2M2ZiYzk0ZWZkYmE3MTMyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DAC2LdV8cwUEIbaaLVS_S5lySEKejVe9-pUn_MeebHo"> <img alt="Screenshot 2025-07-23 121004" width="1919" height="913" src="https://private-user-images.githubusercontent.com/146310937/469600436-b3a4a9ea-d5a4-4889-97c6-1ede4ae4d20e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAwNDM2LWIzYTRhOWVhLWQ1YTQtNDg4OS05N2M2LTFlZGU0YWU0ZDIwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOWZkN2MwN2Y5ZmU2ODVjZDkwMDkxMTc4ZmQ3OWRkZTA2Y2Q2ZmUwOGQwNzlmYzllYmU5MDdjYTA3Y2MzYjU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jkQdt1kZ5Sxl_0pj64RcZufMgatEL-4z8_uijNZZUqQ">
> After:
> 
> <img alt="Screenshot 2025-07-23 121823" width="1919" height="1012" src="https://private-user-images.githubusercontent.com/146310937/469601202-f3198e5a-13e5-4819-8fca-bd9cfde825a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAxMjAyLWYzMTk4ZTVhLTEzZTUtNDgxOS04ZmNhLWJkOWNmZGU4MjVhMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZDY2Y2I4YmJlZWE2MTIwNWYyNTFmMDc0MTM0NmVmNzY1NzlkNGE0ZGQwNTVjYTlkZmY2ZmY2NWEyZjkwZWE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Rff0LgnsAzErh5UsM-pGjCbs-YsqKBIWrwK0B2YnP1Y"> <img alt="Screenshot 2025-07-23 121805" width="1916" height="932" src="https://private-user-images.githubusercontent.com/146310937/469601338-d0d37c06-0947-4970-9d0f-f189834d33a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAxMzM4LWQwZDM3YzA2LTA5NDctNDk3MC05ZDBmLWYxODk4MzRkMzNhNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YzBkZDljYTM0NjgxMGRiZmYxOTdiMGQ5ZjFkZGU4ODg0MDYxNzc2NGE0NjU0ZTQ3ZTI3NTk5NmRlMmUzMDJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xPXFa3eTSvZ6RSUAIjy0ZT4u-UnGUBMZragaya_BfZs">
> **Fixed grammatical errors( found ---> find):**
> 
> Before:
> 
> <img alt="Screenshot 2025-07-23 121032" width="1919" height="954" src="https://private-user-images.githubusercontent.com/146310937/469602023-e0f4cb9c-1136-4c6f-9208-ff83afc0673b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAyMDIzLWUwZjRjYjljLTExMzYtNGM2Zi05MjA4LWZmODNhZmMwNjczYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2ZiMGViNGFmNjIxNjA0ZTZlNTdmZDdjNTc3MDUyNDhmNmE4YWM4NGM0OTMxNTEyOGZlNTlmYjIzNzNlYzM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Fb03CuSy64qz3Qunjj_2z6lIEhum2kWMe3R7mhFHkd4">
> After:
> 
> <img alt="Screenshot 2025-07-23 121748" width="1919" height="1011" src="https://private-user-images.githubusercontent.com/146310937/469602386-73546994-f603-44f6-8394-c62f3cafa670.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyNjEwOTQsIm5iZiI6MTc1MzI2MDc5NCwicGF0aCI6Ii8xNDYzMTA5MzcvNDY5NjAyMzg2LTczNTQ2OTk0LWY2MDMtNDRmNi04Mzk0LWM2MmYzY2FmYTY3MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QwODUzMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZDczZjk5Njk5MTliY2YxOWI2MjE2NjNmZjJhYWU0YTNmNjU1NmM1ZDg1MmRjY2IyNDczN2NlMTE0NGM2ZjliJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oNwCSyM9QQTnFoC_ioT5OW1CPJrxxSsA_Qoxamu3dLI">
> Let me know if anything else needs to be updated. Thank you!

I preferred the previous version of the masthead section, so could you please revert that change? Also, the updated navbar looks great just one small tweak: in the second image, the close (×) icon blends into the navbar due to similar coloring. Could you adjust that so it’s more visible? Thanks!

rajasmitaa commented:
sure but could you please clarify by what you meant about the masthead section? like are you talking about the layout? if yes then no worries im not changing the designs. I corrected a grammatical error. it was written "you can found her on instagram", I changed it to "you can find her on instagram". do you want me to revert that?
